MPR#7647: emphasize ocaml.org links in readme

yawaramin opened the pull request:
Link to appropriate pages in ocaml.org for the homepage, getting OCaml,
documentation, and discussion fora.

gasche commented on Changes:
```diff
@@ -24,6 +24,8 @@ be mentioned in the 4.06 section below instead of here.)
 
 ### Manual and documentation:
 
+- PR#7647: emphasize ocaml.org website and forum in README
```
If you want a Change entry, you should have a proper credit line like other entries.

yawaramin replied:
Ah, OK, done. I figured since this will be a user-facing change (?), it should have an entry.

gasche replied:
Yes, I agree that this deserves a Change entry!

gasche reviewed (commented):
In general I agree with the spirit of the change, but I don't agree with removing all mentions of the caml-list@inria.fr mailing-list from the README. The current status remains that the mailing-list is the sanctioned forum for community discussion, and the Discuss website is more of an experiment.

I think you should keep the part on the caml-list (the description, the adress, the archives), and mention Discuss only second. I would personally also keep the IRC channel in the description (it has a many-years track record of being actively used), before you redirect to the `/community/` webpage for more details and other places.

yawaramin committed:
MPR#7647: emphasize ocaml.org links in readme

Link to appropriate pages in ocaml.org for the homepage, getting OCaml,
documentation, and discussion fora.

yawaramin commented:
@gasche thanks for the review, I've reworded the section to add back the caml-list as the main forum, and also specifically pointed out the `#ocaml` channel.

yawaramin commented:
Bumping this as I believe it will be genuinely helpful to newcomers.

gasche commented on README.adoc:
```diff
@@ -75,45 +75,37 @@ Windows, see link:README.win32.adoc[].
-
-The community also maintains the Web site http://ocaml.org, with tutorials
-and other useful information for OCaml users.
+http://ocaml.org/
```
Can we give a more precise link to where the OCaml manual is? For caml.inria.fr, that would be <http://caml.inria.fr/pub/docs/manual-ocaml/> (which points to versions in other formats as well). What is the corresponding ocaml.org link? 

(<http://caml.inria.fr/pub/distrib/ocaml-4.05/> would also be a natural choice of URL but it has the downside of being specific to an OCaml version, and we don't want to have to change the URL on each release; if we kept pointing at caml.inria.fr it would make sense to add a distrib/ocaml-latest directory. We should be careful of having the same release-robustness property on ocaml.org.)

yawaramin replied:
I didn't add more detail because the previous link was also to the old homepage, so it would take a user a similar number of clicks to reach the manual. If you feel it would help though, I can point directly to the manual, which of course remains at http://caml.inria.fr/pub/docs/manual-ocaml/

gasche replied:
Yes, please point to the manual directly (on caml.inria.fr or ocaml.org).

gasche commented on README.adoc:
```diff
@@ -75,45 +75,37 @@ Windows, see link:README.win32.adoc[].
-
-Messages to the list should be sent to:
+The OCaml mailing list is the official forum for users of the OCaml
+implementations developed at INRIA. You can email it at
```
I would propose:

> The OCaml mailing list is the longest-running forum for OCaml users. You can email it at

yawaramin commented:
OK, branch updated with results of latest review.

gasche commented on Changes:
```diff
@@ -32,7 +32,9 @@ be mentioned in the 4.06 section below instead of here.)
 ### Manual and documentation:
 
-### Compiler distribution build system:
+- PR#7647: emphasize ocaml.org website and forum in README (Yawar Amin)
```
Sorry for the endless nitpicking, but:
- could you include `GPR#1384` as well?
- the credit line should be on its own line

yawaramin replied:
No worries, done and done :-)

gasche commented:
Thanks! Merged.
